addons/mail/static/src/backend_components/many2one_avatar_user/many2one_avatar_user.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.Many2oneAvatarUser" owl="1">
+        <div class="o_field_many2one_avatar cursor-pointer">
+            <div class="o_m2o_avatar d-flex flex-row">
+                <img t-if="props.value" t-att-src="url" t-att-alt="value" t-on-click="openChat" class="Many2oneAvatarUser rounded-circle o_object_fit_cover me-2" />
```

Comment:
There was a `stopPropagation()` so that multi-record views (e.g. `ListView`) do not open the record.
Same question as before: are you sure this field is only used in `FormView`?